src/sagemaker/workflow/pipeline_context.py

```diff
@@ -93,13 +136,42 @@ def runnable_by_pipeline(run_func):
     The job will be started during pipeline execution.
     """
 
+    def _restructure_model_step_args(context: dict) -> dict:
+        """Restructure the pipeline session context arguments for a model step
+
+        Args:
+            context (dict): The pipeline session context to be restructured.
+
+        Returns:
+            dict: the restructured context arguments of a model step.
+        """
+        from sagemaker.workflow.model_step import (
+            MODEL_STEP_ARGS,
+            NEED_RUNTIME_REPACK,
+            CREATE_MODEL_PACKAGE_REQUEST,
+            CREATE_MODEL_REQUEST,
+        )
```

Comment:
Let's use a formal class _ModelStepArguments to capture the information.

Reply:
Added in the new commit